oap-server/server-starter/src/main/resources/otel-rules/apisix.yaml

```diff
@@ -0,0 +1,81 @@
+#    "P2D"       -- parses as "2 days" (where a day is 24 hours or 86400 seconds)
+#    "P2DT3H4M"  -- parses as "2 days, 3 hours and 4 minutes"
+#    "P-6H3M"    -- parses as "-6 hours and +3 minutes"
+#    "-P6H3M"    -- parses as "-6 hours and -3 minutes"
+#    "-P-6H+3M"  -- parses as "+6 hours and -3 minutes"
+# </pre>
+filter: "{ tags -> tags.job_name == 'apisix-monitoring' }" # The OpenTelemetry job name
+expPrefix:  tag({tags -> tags.host_name = 'apisix::' + tags.service_instance_id})
+expSuffix:
+metricPrefix: meter_apisix
+metricsRules:
+  # Service
+  - name: sv_http_connections
+    exp: apisix_nginx_http_current_connections.sum(['state','host_name']).service(['host_name'] , Layer.APISIX)
+  - name: shared_dict_capacity_bytes
+    exp: apisix_shared_dict_capacity_bytes.sum(['name','host_name']).service(['host_name'] , Layer.APISIX)
+  - name: shared_dict_free_space_bytes
+    exp: apisix_shared_dict_free_space_bytes.sum(['name','host_name']).service(['host_name'] , Layer.APISIX)
+  - name: sv_etcd_indexes
+    exp: apisix_etcd_modify_indexes.sum(['key','host_name']).service(['host_name'] , Layer.APISIX)
+  - name: sv_etcd_reachable
+    exp: apisix_etcd_reachable.downsampling(LATEST).service(['host_name'] , Layer.APISIX)
+  - name: http_requests
+    exp: apisix_http_requests_total.rate('PT1M').service(['host_name'] , Layer.APISIX)
+    # Not match any route
+    # Refer to  https://apisix.apache.org/docs/apisix/plugins/prometheus/
+  - name: sv_bandwidth_unmatched
+    exp: apisix_bandwidth.tagEqual('route','').tagEqual('node','').sum(['type','host_name']).rate('PT1M').service(['host_name'] , Layer.APISIX)
+  - name: sv_http_status_unmatched
+    exp: apisix_http_status.tagEqual('route','').tagEqual('node','').sum(['code','host_name']).rate('PT1M').service(['host_name'] , Layer.APISIX)
+  - name: sv_http_latency_unmatched
+    exp:  apisix_http_latency.tagEqual('route','').tagEqual('node','').sum(['type','le','host_name']).histogram().histogram_percentile([50,70,90,99]).service(['host_name'] , Layer.APISIX)
+   # Match a route
+  - name: sv_bandwidth_matched
+    exp: apisix_bandwidth.tagNotEqual('route','').tagNotEqual('node','').sum(['type','host_name']).rate('PT1M').service(['host_name'] , Layer.APISIX)
+  - name: sv_http_status_matched
+    exp: apisix_http_status.tagNotEqual('route','').tagNotEqual('node','').sum(['code','host_name']).rate('PT1M').service(['host_name'] , Layer.APISIX)
+  - name: sv_http_latency_matched
+    exp:  apisix_http_latency.tagNotEqual('route','').tagNotEqual('node','').sum(['type','le','host_name']).histogram().histogram_percentile([50,70,90,99]).service(['host_name'] , Layer.APISIX)
+
+  # Instance
+  # Reorganization metrics which has `node` label to instance level (node as instance name)
+  - name: node_http_status
+    exp: apisix_http_status.tagNotEqual('node','').sum(['code','host_name','node']).rate('PT1M').instance(['host_name'],['node'], Layer.APISIX)
```

Comment:
`PT1M` should be `per minute`? I can see you use `per second`?

Reply:
https://github.com/apache/skywalking/blob/5ae4c125f8cbeb7c53fed4269ca110eeac9aa661/oap-server/analyzer/meter-analyzer/src/main/java/org/apache/skywalking/oap/meter/analyzer/dsl/SampleFamily.java#L293-L295

`PT1M` means time range ， but the calculated result is `per second`

Reply:
Let's use `Increasement per second` as the unit? Decreasement should be a negative value, right?

Reply:
```
# HELP apisix_http_status HTTP status codes per service in Apisix
# TYPE apisix_http_status counter
apisix_http_status{code="200",route="1",matched_uri="/hello",matched_host="",service="",consumer="",node="127.0.0.1"} 4
apisix_http_status{code="200",route="2",matched_uri="/world",matched_host="",service="",consumer="",node="127.0.0.1"} 4
apisix_http_status{code="404",route="",matched_uri="",matched_host="",service="",consumer="",node=""} 1
```
apisix_http_status is `counter` , means the value never decrease excepting  restarting server . it 's  prometheus spec .

Reply:
OK, then please only change the unit for all similar metrics.

Reply:
similar metrics ? you  mean change other metrics unit ?  

<img width="629" alt="image" src="https://user-images.githubusercontent.com/3917424/193858109-07b7a3dd-9b65-493b-bdb5-35028e58bdca.png">

Reply:
How about changing unit to `query per second`  , like QPS

Reply:
Then it is better to be `requests per second`.

Another question, what is the difference between this and `HTTP Request Trend`?

Reply:
<img width="1459" alt="image" src="https://user-images.githubusercontent.com/3917424/193869219-6eba41ca-385f-4231-84fa-498c1946539b.png">